### ISSUE TITLE
Vector Node metrics  (via prometheus)

### ISSUE THREAD

**Issue by lsampras** [juspay/hyperswitch#5031]

**Comment by ShivanshMathurJuspay** [comment]
Hey @anaypurohit0907 
Yes issue is still up for contribution

**Comment by anaypurohit0907** [comment]
hi @ShivanshMathurJuspay  i would like to contribute to this, but i have never worked with grafana. please guide me through this .

**Comment by ShivanshMathurJuspay** [comment]
Sure
you can refer to how to create a dashboard on grafana with adding sources ( prometheus in this case )
https://grafana.com/docs/grafana/latest/getting-started/build-first-dashboard/

**Comment by ShivanshMathurJuspay** [comment]
Hey @anaypurohit0907 
I hope the above reference is helpful

**Comment by anaypurohit0907** [comment]
yes, its helpfull, thanks, its taking longer than usual as i have exams, sorry for the delay
